int8 joint_no
---
bool result